README.md

```diff
@@ -48,6 +48,10 @@ nile init
 pip install openzeppelin-cairo-contracts
 ```
 
+> ⚠️ Warning! ⚠️
+<br>
```

Comment:
It would be good to add the actual warning inside the block? looks better for me to catch the user's attention with the warning explanation too.

Reply:
<img width="902" alt="Screenshot 2022-11-25 at 13 01 00" src="https://user-images.githubusercontent.com/26070224/203982215-b7788721-ab17-41ce-99c2-f3fa33b83c67.png">

Reply:
Agreed! And that was the idea, but the preview in VSCode misled me. All fixed :)